dotCMS/src/main/java/com/dotmarketing/osgi/GenericBundleActivator.java

```diff
@@ -164,6 +171,13 @@ protected void publishBundleServices ( BundleContext context ) throws Exception
     }
 
     /**
+     * @deprecated Use {@link #overrideClasses(BundleContext)} instead
+     */
+    protected void publishBundleServices ( BundleContext context ) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)